pkg/export/export.go

```diff
@@ -936,6 +936,7 @@ func (b batch) send(
 
 			samplesPerRPCBatch.Observe(float64(len(l)))
 
+			fmt.Println("Send", l)
```

Comment:
Leftover debugging, unless you may want to use a logger here?

Reply:
Ups!